community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/RelationshipGroupCursor.java

```diff
@@ -78,4 +78,6 @@ default int totalCount()
     long incomingReference();
 
     long loopsReference();
+
+    int type();
```

Comment:
Maybe you missed `relationshipLabel`? I agree this is better, let's change to this.